collect_app/src/main/java/org/odk/collect/android/tasks/FormLoaderTask.java

```diff
@@ -124,8 +126,8 @@ protected FECWrapper doInBackground(String... path) {
         final File formXml = new File(formPath);
 
         // set paths to /sdcard/odk/forms/formfilename-media/
-        final String formFileName = formXml.getName().substring(0, formXml.getName().lastIndexOf("."));
-        final File formMediaDir = new File(formXml.getParent(), formFileName + "-media");
+        final String formFileName = getFormFileName(formXml);
```

Comment:
Maybe take the chance to rename this variable to something more precise, like formFileNameBase (since it's not the full filename, but the filename after cutting the extension)?